### DISCUSSION TITLE
[feature] option to delete a request

### DISCUSSION THREAD

**Discussion by diegoulloao** [darrenburns/posting#24] [Ideas]
I've been enjoying `posting`, in testing phase now, one thing it catches my attention is that there's no option to delete a request, sometimes I accidentally create a new request and I'd like to remove it somehow from the TUI.

<img width="480" alt="Screenshot 15" src="https://github.com/darrenburns/posting/assets/45423661/3b2b4412-9bb3-4ea6-9654-63559e0f2764">

---

Suggestion: `ctrl+d` for delete a request.
(with confirm box dialog)

<img width="614" alt="Screenshot 16" src="https://github.com/darrenburns/posting/assets/45423661/e18b712b-7b51-4fa3-a8ae-45eb9eb6e63e">

It could be included at the bottom as the rest of shortcuts.

**Comment by darrenburns** [comment]
This was added in a recent release :)
